### PR TITLE
🤯 What if routers had categories that mapped to exits

### PR DESCRIPTION
**Pull request by rowanseymour**
Addresses a need in the future for multiple categories to map to a single "physical" exit

### CODE REVIEW

File: cmd/docgen/templates/flows.md [hunk 1]
```diff
@@ -75,14 +75,14 @@ An exit consists of:
...
-to true, then the `default_exit_uuid` will be used otherwise flow execution will stop.
+to true, then the `default_category_uuid` will be used otherwise flow execution will stop.
 
 A switch router may also define a `result_name` parameters which will save the result of the case which evaluated as true.
 
 A switch router consists of:
 
  * `operand` the expression which will be evaluated against each of our cases
- * `default_exit_uuid` the uuid of the default exit to take if no case matches (optional)
+ * `default_category_uuid` the uuid of the default exit to take if no case matches (optional)
```
Comment (nicpottier) [inline cmd/docgen/templates/flows.md]: `default category` not `default exit`

File: docs/md/flows.md [hunk 2]
```diff
@@ -75,14 +75,14 @@ An exit consists of:
...
-to true, then the `default_exit_uuid` will be used otherwise flow execution will stop.
+to true, then the `default_category_uuid` will be used otherwise flow execution will stop.
 
 A switch router may also define a `result_name` parameters which will save the result of the case which evaluated as true.
 
 A switch router consists of:
 
  * `operand` the expression which will be evaluated against each of our cases
- * `default_exit_uuid` the uuid of the default exit to take if no case matches (optional)
+ * `default_category_uuid` the uuid of the default exit to take if no case matches (optional)
```
Comment (nicpottier) [inline docs/md/flows.md]: same

File: cmd/flowmigrate/main.go [hunk 3]
```diff
@@ -17,17 +17,16 @@ import (
 )
 
 func main() {
-	var includeUI, collapseExits bool
```
Comment (rowanseymour) [inline cmd/flowmigrate/main.go]: always collapse exits now

**Comment by rowanseymour** [comment]
@nicpottier think this is ready now 😬

**Pull request merged**